src/nested_pandas/nestedframe/io.py

```diff
@@ -187,26 +195,28 @@ def _read_parquet_into_table(
     # NOTE: the test for _is_local_dir is sufficient, because we're
     # preserving a path to pq.read_table, which can read local
     # directories, but not remote directories. Remote directories
-    # are supported separately via _read_parquet_directory
+    # are supported separately via _read_parquet_directory.
+    # We don't support HTTP "directories", because 1) calling .is_dir()
+    # may be very expensive, because it downloads content first,
+    # 2) because .iter_dir() is likely to return a lot of "junk"
+    # besides of the actual parquet files.
```

Comment:
Great comment, thank you! Maybe it belongs up by the `NO_ITERDIR_FILESYSTEMS` constant?  (But here is great, too.)